subgraph/src/ids/ids.ts

```diff
@@ -0,0 +1,34 @@
+import {Address, BigInt, Bytes} from '@graphprotocol/graph-ts';
+
+/**
+ * Returns a valid entity ID based on the provided address.
+ *
+ * @export
+ * @param {Address} address
+ * @return {*}  {string}
+ */
+export function generateEntityIdFromAddress(address: Address): string {
+  return address.toHexString();
+}
+
```

Comment:
@Michael-A-Heuer for having a second opinion do we need theses functions really?

Reply:
To me, it makes sense to have a unified way to generate entity IDs from addresses. If this changes, we only need to change it in one place.
Could also help to handle conversion to lower-case

Reply:
Tested and the conveersion to lowercase is already done by the toHexString